content/case-studies/sourcegraph.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Sourcegraph
+description: |
+    Pulumi helps to supercharge Kubernetes deployments at Sourcegraph, ensuring their team has continuous access to the latest builds.
+meta_desc: |
+    Pulumi helps to supercharge Kubernetes deployments at Sourcegraph, ensuring their team has continuous access to the latest builds.
+
+customer_name: Sourcegraph
+customer_logo: /logos/customers/sourcegraph_logo.png
```

Comment:
This is fine, but I'd rather use an SVG for this, and there appears to be one on the Sourcegraph site at https://about.sourcegraph.com/sourcegraph/logo.svg. If you're okay with it, I'll push a PR onto this branch that uses that SVG and replaces the one on the home page also. (It needs a quick adjustment for size.)